Tests/test_AlignIO.py

```diff
@@ -186,6 +188,28 @@ def simple_alignment_comparison(alignments, alignments2, format):
                        "'%s' vs '%s'" % (r1.id, r2.id)
     return True
 
+#Check Phylip files reject duplicate identifiers.
+def check_phylip_reject_duplicate():
+    """
```

Comment:
Never mind, I clicked on a wrong link and though this was a live pull request. Sorry.